azuredeploy.json

```diff
@@ -787,6 +848,7 @@
             "redisDeploySwitch": "[parameters('redisDeploySwitch')]",
             "redisDns": "[concat('redis-',variables('resourceprefix'),'.redis.cache.windows.net')]",
             "resourcesPrefix": "[variables('resourceprefix')]",
+            "searchType": "[parameters( 'searchType')]",
```

Comment:
Sorry for nitpicking, but let's remove the blank space between the opening paren and the argument.